scripts/ci/generate-changelog

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -ex
+
+MAJOR_MINOR_MATCH=$(echo $TAG_NAME | grep -Eo "^v?(0|[1-9]\d*)\.(0|[1-9]\d*)")
+TAGS=($(git tag -l "$MAJOR_MINOR_MATCH*" --sort=-version:refname))
+PREVIOUS_TAG=${TAGS[1]}
+
+if [[ -z $PREVIOUS_TAG ]]
+then
+  exit
+fi
+
+COMMITS=$(git log $PREVIOUS_TAG..$TAG_NAME --pretty=format:"%H")
+JIRA_PREFIX="(DCOS|DCOS_OSS|MARATHON)"
+MARKDOWN=""
+
+for COMMIT in $COMMITS; do
+
+  COMMIT_BODY=$(git log -1 ${COMMIT})
+  COMMIT_BODY_ONELINE=$(echo $COMMIT_BODY)
+  COMMIT_COMMENT=$(git log --format=%B -n 1 ${COMMIT})
+
+  if [[ $COMMIT_BODY_ONELINE =~ Closes\ ${JIRA_PREFIX}-[0-9]* ]]; then
+    TASK_NUMBER=$(echo $COMMIT_BODY_ONELINE | grep -Eo "${JIRA_PREFIX}-[0-9]*")
+    TASK_DESCRIPTION=$(echo "${COMMIT_COMMENT}" | grep -Eo -m1 ".*:(.*)" | cut -d: -f2)
+
+    MARKDOWN+="[${TASK_NUMBER}](https://jira.mesosphere.com/browse/${TASK_NUMBER}): ${TASK_DESCRIPTION}"
```

Comment:
Wow, this requires a lot of discipline for our commit messages ^^

Reply:
It simply requires us to follow the commitlint format, which is already enforced as post-commit
 script 😸